src/css/global.scss

```diff
@@ -1,4 +1,82 @@
 @import '~css-reset-and-normalize';
+@import './core/constants';
+@import './core/mixins';
+
+$basic-blue: #bcc8dc;
+$basic-grey: #edeff5;
+$dark-red: #d93347;
+$dark-blue: #314a74;
```

Comment:
Why did you copy the color constants from `src/css/core/contants.scss` to this file? I thought they had been imported by `@import './core/constants';`.